public/app/features/plugins/extensions/utils.tsx

```diff
@@ -408,3 +411,145 @@ export const openAppInSideview = (pluginId: string) => sidecarService.openApp(pl
 export const closeAppInSideview = (pluginId: string) => sidecarService.closeApp(pluginId);
 
 export const isAppOpened = (pluginId: string) => sidecarService.isAppOpened(pluginId);
+
+// Comes from the `app_mode` setting in the Grafana config (defaults to "development")
+// Can be set with the `GF_DEFAULT_APP_MODE` environment variable
+export const isGrafanaDevMode = () => config.buildInfo.env === 'development';
+
+// Checks if the meta information is missing from the plugin's plugin.json file
+export const isExtensionPointMetaInfoMissing = (extensionPointId: string, pluginContext: PluginContextType) => {
+  const pluginId = pluginContext.meta?.id;
+  const extensionPoints = pluginContext.meta?.extensions?.extensionPoints;
+
+  if (!extensionPoints || !extensionPoints.find((ep) => ep.id === extensionPointId)) {
```

Comment:
nit
```suggestion
  if (!extensionPoints || !extensionPoints.some((ep) => ep.id === extensionPointId)) {
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/93429/commits/d02fbf48df3716e7fdd4d6468de66d23b9ba8976.